ext/pdo_sqlite/sqlite_driver.c

```diff
@@ -294,13 +294,19 @@ static int pdo_sqlite_get_attribute(pdo_dbh_t *dbh, zend_long attr, zval *return
 static bool pdo_sqlite_set_attr(pdo_dbh_t *dbh, zend_long attr, zval *val)
 {
 	pdo_sqlite_db_handle *H = (pdo_sqlite_db_handle *)dbh->driver_data;
+	zend_long lval;
+
+	/* As all attributes use an integer check this in advance */
```

Comment:
Not unknown attributes though.

Reply:
Indeed, don't know how I'm missing those case